src/main/java/uk/gov/ons/census/casesvc/service/CaseProcessor.java

```diff
@@ -58,6 +53,16 @@ public Case saveCase(CreateCaseSample createCaseSample) {
     return caze;
   }
 
+  public int getUniqueCaseRef() {
```

Comment:
why is this public?

Reply:
It's reused by individual response request